stats/rpc_constants.textpb

```diff
@@ -0,0 +1,770 @@
+# Constants for collecting rpc stats.
+rpc_constants {
+  # Census defined rpc client MeasurementDescriptors.
+  measurement_descriptor {
+    name: "/rpc/client/error_count"
+    description: "RPC Errors"
+    unit: {
+      power10: 0
```

Comment:
No need to specify 0 values for power10 (this is the default if not explictly set).